api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/ParameterNameConverter.scala

```diff
@@ -19,6 +19,8 @@ package ai.h2o.sparkling.api.generation.common
 
 object ParameterNameConverter {
   val h2oToSWExceptions: Map[String, String] = Map(
+    "response_column" -> "labelCol",
+    "weights_column" -> "weightCol",
```

Comment:
In the conversion rule bellow for Columns, is is able to capture `fold_column` ? The rule has capital `C` in `Column`

Reply:
The conversion rule is applied on a capitalized sparkling water name. Here is the conversion logic:
```
  def convertFromH2OToSW(parameterName: String): String = {
    val parts = parameterName.split("_")
    val capitalizedParts = parts.head +: parts.tail.map(_.capitalize)
    val regularValue = conversionRules.foldLeft(capitalizedParts.mkString) {
      case (input, (from, to)) => input.replace(from, to)
    }
    h2oToSWExceptions.getOrElse(parameterName, regularValue)
  }
```

Reply:
Thanks!